CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [\#10311](https://github.com/cosmos/cosmos-sdk/pull/10311) Adds cli to use tips transactions. It adds an `--aux` flag to all CLI tx commands to generate the aux signer data (with optional tip), and a new `tx aux-to-fee` subcommand to let the fee payer gather aux signer data and broadcast the tx
 * [\#10430](https://github.com/cosmos/cosmos-sdk/pull/10430) ADR-040: Add store/v2 `MultiStore` implementation
 * [\#10614](https://github.com/cosmos/cosmos-sdk/pull/10614) Support in-place migration ordering
+* [\#10893](https://github.com/cosmos/cosmos-sdk/pull/10893) Add CLI support for withdraw proposal.
```

Comment:
probably best to be a bit more descriptive, ie saying this is for **group** proposals 
```suggestion
* [\#10893](https://github.com/cosmos/cosmos-sdk/pull/10893) Add CLI support for withdrawing group proposal.
```